spec/acceptance/manage/bzr_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'vcsrepo::manage::bzr' do
+  case os[:family] # bzr not present on RHEL or SLES
+  when 'Debian', 'Archlinux'
+    let(:pp) { 'include vcsrepo::manage::bzr' }
+
+    it 'applies idempotently' do
+      idempotent_apply(pp)
+    end
+
+    it 'installs bzr' do
+      expect(package('bzr')).to be_installed
+    end
+  else
+    0
```

Comment:
I'd put the comment here
```suggestion
    # bzr not present on RHEL or SLES
```